app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -126,6 +123,13 @@
     private ImageView arcPointer;
     private LinearLayout infoLayout;
 
+    private LinearLayout touchView;
+    private WindowManager.LayoutParams touchViewParams;
+    private Handler screenScanHandler;
+    private Runnable screenScanRunnable;
+    private final int screenScreenDelay = 750;
```

Comment:
This was the whole point of adding this variable, but I forgot to use it. Did you test it? How well did it work for you?